subsys/bluetooth/mesh/access.c

```diff
@@ -33,6 +33,11 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_mesh_access);
 
+/* 20 - 50ms */
+#define RANDOM_DELAY_SHORT 30
```

Comment:
I guess it is worth to add at least layer prefix or better `BT_MESH_`. Of course, they are local, but naming is very common. It might coincide with third party definitions via headers.
Up to you.

Reply:
We don't do this for internal variables, so no, I won't change this.